go.mod

```
@@ -1,4 +1,4 @@
-module github.com/StefMa/hugo-fresh
+module github.com/selkhateeb/hugo-fresh
```

This should be reverted 😇 
```suggestion
module github.com/StefMa/hugo-fresh
```